### PR TITLE
Rev 3

### PR DESCRIPTION
**Pull request by lucywowen**

### CODE REVIEW

File: paper/main.tex [hunk 1]
```diff
@@ -338,38 +338,37 @@ \subsubsection*{Recovering dynamic higher-order correlations}
...
 better able to recover dynamic first-order correlations than
-second-order correlations.  This is because errors in our \textit{estimation}
+second-order correlations.  This is because errors in our estimation
 procedure at lower orders necessarily propagate to higher orders
 (since lower-order correlations are used to estimate higher-order
 correlations).  Conversely, when the correlations were particularly
-\textit{un}stable (random datasets), we better recovered second-order
-correlations.  This is because noise in our \textit{data generation}
+unstable (random datasets), we better recovered second-order
+correlations.  This is because noises in our data generation
```
Comment (jeremymanning) [inline paper/main.tex:346]: noises --> noise (revert change)

File: paper/main.tex [hunk 2]
```diff
@@ -338,38 +338,37 @@ \subsubsection*{Recovering dynamic higher-order correlations}
...
   \centering
   \includegraphics[width=\textwidth]{figs/higher_order_sims}
-  \caption{\textbf{Recovery of simulated first-order and second-order dynamic correlations.}  Each panel displays the average correlations
+  \caption{Recovery of simulated first-order and second-order dynamic correlations.  Each panel displays the average correlations
     between the vectorized upper triangles of the recovered
     first-order and second-order correlation matrices and the true
     (simulated) first-order and second order correlation matrices at each
     timepoint and for each synthetic dataset.  (The averages are taken across 100 different
-    randomly generated synthetic datasets of each given category, each
-    with $K = 10$ features and $T = 300$ timepoints.)
```
Comment (jeremymanning) [inline paper/main.tex:359]: keep?  i think one of the reviewers asked for these details in our prior revision, right?

File: paper/main.tex [hunk 3]
```diff
@@ -415,22 +414,22 @@ \subsubsection*{Recovering dynamic higher-order correlations}
...
   fMRI data}
 We used across-participant temporal decoders to identify cognitively
-relevant neural patterns in fMRI data (see \textit{Forward inference
-  and decoding accuracy}).  The dataset we examined~\citep[collected
+relevant neural patterns in fMRI data (see Forward inference
+  and decoding accuracy).  The dataset we examined~\citep[collected
 by][]{SimoEtal16} comprised four experimental conditions that exposed
 participants to stimuli that varied systematically in how cognitively
-engaging they were.  The \textit{intact} experimental condition had
+engaging they were.  The intact experimental condition had
```
Comment (jeremymanning) [inline paper/main.tex:421]: the italics here were used to describe figure labels-- i'd change this line to:

"engaging they were.  The intact experimental condition (intact) had..."

File: paper/main.tex [hunk 4]
```diff
@@ -415,22 +414,22 @@ \subsubsection*{Recovering dynamic higher-order correlations}
...
 participants listen to an audio recording of a 10-minute story.  The
-\textit{paragraph}-scrambled experimental condition had participants
+paragraph-scrambled experimental condition had participants
```
Comment (jeremymanning) [inline paper/main.tex:423]: change to:

"paragraph-scrambled experimental condition (paragraph) had participants..."

File: paper/main.tex [hunk 5]
```diff
@@ -415,22 +414,22 @@ \subsubsection*{Recovering dynamic higher-order correlations}
...
 listen to a temporally scrambled version of the story, where the
 paragraphs occurred out of order (but where the same total set of
 paragraphs were presented over the full listening interval).  All
 participants in this condition experienced the scrambled paragraphs in
-the same order.  The \textit{word}-scrambled experimental condition
+the same order.  The word-scrambled experimental condition
```
Comment (jeremymanning) [inline paper/main.tex:428]: change to:

the same order.  The word-scrambled experimental condition (word)...

File: paper/main.tex [hunk 6]
```diff
@@ -415,22 +414,22 @@ \subsubsection*{Recovering dynamic higher-order correlations}
...
 had participants listen to a temporally scrambled version of the story
 where the words in the story occurred in a random order.  All
 participants in the word condition experienced the scrambled words in
-the same order.  Finally, in a \textit{rest} experimental condition,
+the same order.  Finally, in a rest experimental condition,
```
Comment (jeremymanning) [inline paper/main.tex:432]: change to:

the same order.  Finally, in a rest experimental condition (rest),

File: paper/main.tex [hunk 7]
```diff
@@ -440,20 +439,20 @@ \subsection*{Cognitively relevant dynamic high-order correlations in
...
   \centering
   \includegraphics[width=\textwidth]{figs/decode_level}
-  \caption{\textbf{Across-participant timepoint decoding accuracy varies with
-      correlation order and cognitive engagement.}
-    \textbf{a.~Decoding accuracy as a function of order: PCA.}
-    \textit{Order} ($x$-axis) refers to the maximum order of dynamic
+  \caption{Across-participant timepoint decoding accuracy varies with
+      correlation order and cognitive engagement.
+    a.~Decoding accuracy as a function of order: PCA.
+    Order ($x$-axis) refers to the maximum order of dynamic
```
Comment (jeremymanning) [inline paper/main.tex:445]: Change to:

``Order'' ($x$-axis) refers to the maximum order of dynamic

File: paper/main.tex [hunk 8]
```diff
@@ -718,19 +718,6 @@ \section*{Discussion}
...
-dynamic correlations.  This means that our measures of which
-higher-order patterns are reliable and stable across experimental
-conditions are partially confounded with the stability of lower-order
-patterns.  Prior work suggests that the stability of what we refer to
-here as first-order dynamics likely varies across the experimental
-conditions we examined~\citep{SimoEtal16}.  Therefore a caveat to our
-claim that richer stimuli evoke more stable higher-order dynamics is
-that our approach assumes that those high-order dynamics reflect
-relations or interactions between lower-order features.
-
```
Comment (jeremymanning) [inline paper/main.tex:733]: can you add this back?

File: paper/main.tex [hunk 9]
```diff
@@ -1145,7 +1132,7 @@ \subsubsection*{Synthetic data: simulating dynamic first-order correlations}
   
   \paragraph*{Ramping.}  We generated a third set of 100 synthetic
   datasets whose underlying correlations changed gradually over time.
-  For each dataset, we constructed two \textit{anchor} covariance
+  For each dataset, we constructed two anchor covariance
```
Comment (jeremymanning) [inline paper/main.tex:1135]: Change to:

For each dataset, we constructed two ``anchor'' covariance

File: paper/main.tex [hunk 10]
```diff
@@ -1218,11 +1205,8 @@ \subsubsection*{Synthetic data: simulating dynamic high-order
 at a given timepoint is $m_n$, then we can generate an order $n-1$
 correlation matrix (for $n > 1$) by taking a draw from
 $\mathcal{N}\left(0, m_n\right)$ and reshaping the resulting vector to
-have square dimensions.  To force the resulting matrix to be
-symmetric, we remove its lower triangle, and replace the lower
-triangle with (a reflected version
-of) its upper triangle.  Intuitively, the resulting re-shaped matrix will look
-like a noisy (but symmetric) version of the template matrix, $m_{n-1}$.  (When
+have square dimensions.  Intuitively, the re-shaped matrix will look
```
Comment (jeremymanning) [inline paper/main.tex:1208]: revert back to old version...

File: paper/main.tex [hunk 11]
```diff
@@ -1235,15 +1219,9 @@ \subsubsection*{Synthetic data: simulating dynamic high-order
...
-correlation estimation procedure, this yielded a 100-dimensional
-timeseries of first-order
-features that could then be used to estimate dynamic second-order
-correlations.  (We chose to use $K=10$ zero-order features for our
-higher order simulations in order to put the accuracy computations
-displayed in Figs.~\ref{fig:synthetic} and~\ref{fig:higher_order_sims}
-on a roughly even footing.)
+and event).  Each timeseries comprised 10 features (dimensions)
+that varied over 300 timepoints for first-order correlations, and
+therefore 100 second-order features.
```
Comment (jeremymanning) [inline paper/main.tex:1225]: revert back to previous version

File: paper/main.tex [hunk 12]
```diff
@@ -1259,7 +1237,7 @@ \subsubsection*{Functional neuroimaging data collected during story
 condition listened to the words in a randomized order), or lay resting
 with their eyes open in the scanner (rest condition; 36
 participants).  Full neuroimaging details may be found in the original
-paper for which the data were collected~\citep{SimoEtal16}.
+paper for which the data were collected~\citep{SimoEtal16}. Procedures were approved by the Princeton University Committee on Activities Involving Human Subjects, and by Western Institutional Review Board (Puyallup, WA). All subjects were native English speakers with normal hearing and provided written informed consent.
```
Comment (jeremymanning) [inline paper/main.tex:1240]: Change to:

Procedures were approved by the Princeton University Committee on Activities Involving Human Subjects, and by *the* Western Institutional Review Board (Puyallup, WA). All subjects were native English speakers with normal hearing and provided written informed consent.

(i.e., at "the" before Western Institutional Review Board)-- I assume you copied this line from Janice's paper?

Comment (jeremymanning) [reply paper/main.tex:1240]: Sorry... I mean Erez's paper

File: paper/main.tex [hunk 13]
```diff
@@ -1471,6 +1449,16 @@ \subsubsection*{Reverse inference}
...
+\section*{Data Availability}
+
+The authors declare that the data supporting the findings of this
+study as well as the source data for this paper are available at \href{https://github.com/ContextLab/timecorr-paper/releases/tag/v0.3}{github.com/ContextLab/timecorr-paper/releases/tag/v0.3}. The fMRI dataset collected by \cite{SimoEtal16} has been made publicly available at
+\href{http://arks.princeton.edu/ark:/88435/dsp015d86p269k}{arks.princeton.edu/ark:/88435/dsp015d86p269k}
+
+\section*{Code Availability}
+
+All of our analysis code may be downloaded from\href{https://github.com/ContextLab/timecorr-paper/releases/tag/v0.3}{github.com/ContextLab/timecorr-paper/releases/tag/v0.3}.
+
```
Comment (jeremymanning) [inline paper/main.tex:1461]: Missing space after "from"

Also should we cite timecorr toolbox?  E.g. you could add something like:

"We have also published a companion Python toolbox that may be downloaded from \href{https://timecorr.readthedocs.io}{timecorr.readthedocs.io}."

File: paper/main.tex [hunk 14]
```diff
@@ -1487,7 +1475,10 @@ \section*{Author contributions}
 Concept: J.R.M.  Implementation: T.H.C., L.L.W.O., and J.R.M.
 Analyses: L.L.W.O. and J.R.M.  Writing: L.L.W.O. and J.R.M.
 
-\bibliographystyle{apacite}
+\section*{Competing interests}
+The authors declare no competing financial interests.
+
+\bibliographystyle{naturemag}
 \bibliography{CDL-bibliography/cdl}
```
Comment (jeremymanning) [inline paper/main.tex:1482]: Once we have everything else together for the paper, you can paste in contents of main.bib directly (replace lines 1481 and 1482 with the contents of main.bib, once the formatting looks good)

File: paper/supplementary_materials.tex [hunk 15]
```diff
@@ -49,30 +52,30 @@
 \includegraphics[width=0.95\textwidth]{figs/decode_levels_kernels}
 \caption{\textbf{Across-participant timepoint decoding accuracy varies with
       correlation order and cognitive engagement across kernels.}
-    \textbf{a.-c.~Timepoint decoding accuracy as a function of order: PCA.} \textit{Order} ($x$-axis) refers to the maximum order of dynamic
+    \textbf{a.-c.~Timepoint decoding accuracy as a function of order: PCA.} Order ($x$-axis) refers to the maximum order of dynamic
     correlations that were available to the classifiers (see
-    \textit{Feature weighting and testing}).  The reported
```
Comment (jeremymanning) [inline paper/supplementary_materials.tex:54]: could you revert back the formatting in the supplemental materials?  i think it's ok to have italics and bold in the supplement

File: paper/supplementary_materials.tex [hunk 16]
```diff
@@ -49,30 +52,30 @@
...
     word: $\frac{1}{300}$; rest: $\frac{1}{400}$).  The error ribbons
-    denote 95\% confidence intervals across cross-validation folds
+    denote 95\% confidence intervals of the mean across cross-validation folds
     (i.e., random assignments of participants to the training and test
     sets).  The colors denote the experimental condition.  Arrows
     denote sets of features that yielded reliably higher (upward
     facing) or lower (downward facing) decoding accuracy than the mean
     of all other features (via a two-tailed $t$-test, thresholded at
-    $p < 0.05$).  The circled values represent
+    $p < 0.05$, exact values are reported in the Source Data file).  The circled values represent
```
Comment (jeremymanning) [inline paper/supplementary_materials.tex:70]: change comma to semicolon

File: paper/supplementary_materials.tex [hunk 17]
```diff
@@ -81,29 +84,29 @@
...
```
Comment (jeremymanning) [inline paper/supplementary_materials.tex:102]: change comma to semicolon

**Review by jeremymanning — CHANGES_REQUESTED** [review]
overall looks great!  could you make the requested changes and then let me know when it's ready to merge?

File: paper/main.tex [hunk 18]
```diff
@@ -234,23 +235,22 @@ \section*{Results}
...
+state scan (see Functional neuroimaging data collected during
+  story listening).  We used the relative decoding accuracies of
 classifiers trained on different sets of neural features to estimate
 which types of features reflected ongoing cognitive processing.
 
-\subsection*{Recovering known dynamic correlations from synthetic
-  data}
-\subsubsection*{Recovering dynamic first-order correlations}
+
+\subsection*{Recovering dynamic first-order correlations}
```
[the remark this answers is not in the document]
Comment (lucywowen) [reply paper/main.tex:253]: added 'known'

File: paper/main.tex [hunk 19]
```diff
@@ -265,40 +265,39 @@ \subsubsection*{Recovering dynamic first-order correlations}
...
+  \caption{Recovering known dynamic first-order correlations
+      from synthetic data.  Each panel displays the average
     correlations between the vectorized upper triangles of the
     recovered correlation matrix at each timepoint and either the true
     underlying correlation at each timepoint or a reference
     correlation matrix.  (The averages are taken across 100 different
-    randomly generated synthetic datasets of each given category, each
-    with $K = 50$ features and $T = 300$ timepoints.)
-    Error ribbons denote 95\% confidence intervals (taken across
+    randomly generated synthetic datasets of the given category.)
```
[the remark this answers is not in the document]
Comment (lucywowen) [reply paper/main.tex:274]: added back detail.  not sure why I took it out in the first place, its not wrong.

File: paper/main.tex [hunk 20]
```diff
@@ -313,18 +312,19 @@ \subsubsection*{Recovering dynamic first-order correlations}
...
 (Fig.~\ref{fig:synthetic}b), a Dirac $\delta$ kernel (infinitely
 narrow) performed best.  This is because, when every timepoint's
-correlations are independent from the correlations at every other
+correlations are independent of the correlations at every other
 timepoint, averaging data over time dilutes the available signal.
 Following a similar pattern, holding kernel shape fixed, narrower
 kernel parameters better recovered randomly varying correlations.
 
-\subsubsection*{Recovering dynamic higher-order correlations}
+\subsection*{Recovering dynamic higher-order correlations}
```
[the remark this answers is not in the document]
Comment (lucywowen) [reply paper/main.tex:320]: added

File: paper/main.tex [hunk 21]
```diff
@@ -313,18 +312,19 @@ \subsubsection*{Recovering dynamic first-order correlations}
...
 Following our approach to evaluating our ability to recover known
 dynamic first-order correlations from synthetic data, we generated an
 analogous second set of synthetic datasets that we designed to exhibit
-known dynamic first-order \textit{and} second-order correlations (see
-\textit{Synthetic data: simulating dynamic higher-order
-  correlations}).  We generated a total of 400 datasets that varied in
+known dynamic first-order and second-order correlations (see
+Synthetic data: simulating dynamic higher-order
```
[the remark this answers is not in the document]
Comment (lucywowen) [reply paper/main.tex:325]: It does still exist.  They said 'We allow only one level of subheadings in the Results section. Please remove secondary subheadings.' They didn't say to remove them in the Methods section, so I left that subsubsection.

File: paper/main.tex (hunk 1 above)
Comment (lucywowen) [reply paper/main.tex:346]: reverted

File: paper/main.tex (hunk 2 above)
Comment (lucywowen) [reply paper/main.tex:359]: yeah I really don't know why I took them out.

File: paper/main.tex (hunk 3 above)
Comment (lucywowen) [reply paper/main.tex:421]: done

File: paper/main.tex (hunk 4 above)
Comment (lucywowen) [reply paper/main.tex:423]: done

File: paper/main.tex (hunk 5 above)
Comment (lucywowen) [reply paper/main.tex:428]: done

File: paper/main.tex (hunk 6 above)
Comment (lucywowen) [reply paper/main.tex:432]: done

File: paper/main.tex (hunk 7 above)
Comment (lucywowen) [reply paper/main.tex:445]: done

File: paper/main.tex (hunk 8 above)
Comment (lucywowen) [reply paper/main.tex:733]: ok I added this entire section back (I think that's what you were asking for???)

One limitation of our approach relates to how noise propagates in our
estimation procedure.  Specifically, our procedure for estimating
high-order dynamic correlations depends on estimates of lower-order
dynamic correlations.  This means that our measures of which
higher-order patterns are reliable and stable across experimental
conditions are partially confounded with the stability of lower-order
patterns.  Prior work suggests that the stability of what we refer to
here as first-order dynamics likely varies across the experimental
conditions we examined~\citep{SimoEtal16}.  Therefore a caveat to our
claim that richer stimuli evoke more stable higher-order dynamics is
that our approach assumes that those high-order dynamics reflect
relations or interactions between lower-order features.

File: paper/main.tex (hunk 9 above)
Comment (lucywowen) [reply paper/main.tex:1150]: done

File: paper/main.tex (hunk 10 above)
Comment (lucywowen) [reply paper/main.tex:1223]: reverted

File: paper/main.tex (hunk 11 above)
Comment (lucywowen) [reply paper/main.tex:1240]: I think I reverted back to what you want, but you might want to check me on that.

File: paper/main.tex (hunk 12 above)
Comment (lucywowen) [reply paper/main.tex:1255]: I did and I changed it to add 'the'

File: paper/main.tex (hunk 13 above)
Comment (lucywowen) [reply paper/main.tex:1477]: spaced

File: paper/main.tex (hunk 14 above)
Comment (lucywowen) [reply paper/main.tex:1498]: okey doke

File: paper/supplementary_materials.tex (hunk 15 above)
Comment (lucywowen) [reply paper/supplementary_materials.tex:54]: added back italics formatting

File: paper/supplementary_materials.tex (hunk 16 above)
Comment (lucywowen) [reply paper/supplementary_materials.tex:70]: done

File: paper/supplementary_materials.tex (hunk 17 above)
Comment (lucywowen) [reply paper/supplementary_materials.tex:102]: done

File: paper/supplementary_materials.tex [hunk 22]
```diff
@@ -81,29 +85,30 @@
 \includegraphics[width=0.95\textwidth]{figs/decode_levels_widths}
 \caption{\textbf{Across-participant timepoint decoding accuracy varies with
       correlation order and cognitive engagement across widths.}
-    \textbf{a.-d.~Timepoint decoding accuracy as a function of order: PCA.} \textit{Order} ($x$-axis) refers to the maximum order of dynamic
+    \textbf{a.-d.~Timepoint decoding accuracy as a function of order:
+      PCA.} \textit{Order} ($x$-axis) refers to the maximum order of dynamic
     correlations that were available to the classifiers (see
-    \textit{Feature weighting and testing}).  The reported
```
Comment (jeremymanning) [inline paper/supplementary_materials.tex:86]: revert italics

File: paper/supplementary_materials.tex [hunk 23]
```diff
@@ -81,29 +85,30 @@
...
+    Feature weighting and testing).  The reported
     across-participant decoding accuracies for kernel widths of \textbf{a. 5},
     \textbf{b. 10}, \textbf{c. 20}, and \textbf{d. 50} are averaged over all
-    kernel shapes (see \textit{Identifying robust decoding
-      results}).  The $y$-values are displayed relative to chance
```
Comment (jeremymanning) [inline paper/supplementary_materials.tex:90]: revert italics

**Review by jeremymanning — CHANGES_REQUESTED** [review]
just a few minor last things!

File: paper/main.tex (hunk 13 above)
Comment (lucywowen) [reply paper/main.tex:1488]: done!  sorry didn't see that last part earlier

File: paper/main.tex [hunk 24]
```diff
@@ -1145,7 +1147,7 @@ \subsubsection*{Synthetic data: simulating dynamic first-order correlations}
   
   \paragraph*{Ramping.}  We generated a third set of 100 synthetic
   datasets whose underlying correlations changed gradually over time.
-  For each dataset, we constructed two \textit{anchor} covariance
+  For each dataset, we constructed two``anchor'' covariance
```
[the remark this answers is not in the document]
Comment (lucywowen) [reply paper/main.tex:1150]: done

File: paper/main.tex [hunk 25]
```diff
@@ -266,39 +266,39 @@ \subsubsection*{Recovering dynamic first-order correlations}
   \centering
   \includegraphics[width=\textwidth]{figs/synthetic_data}
   \caption{\textbf{Recovering known dynamic first-order correlations
-      from synthetic data.}  Each panel displays the average
+       from synthetic data.} Each panel displays the average
     correlations between the vectorized upper triangles of the
     recovered correlation matrix at each timepoint and either the true
     underlying correlation at each timepoint or a reference
     correlation matrix.  (The averages are taken across 100 different
-    randomly generated synthetic datasets of each given category, each
```
[the remark this answers is not in the document]
Comment (lucywowen) [reply paper/main.tex:274]: done for both

File: paper/main.tex [hunk 26]
```diff
@@ -1259,7 +1261,9 @@ \subsubsection*{Functional neuroimaging data collected during story
 condition listened to the words in a randomized order), or lay resting
 with their eyes open in the scanner (rest condition; 36
 participants).  Full neuroimaging details may be found in the original
-paper for which the data were collected~\citep{SimoEtal16}.
+paper for which the data were collected~\citep{SimoEtal16}. Procedures
+were approved by the Princeton University Committee on Activities
+Involving Human Subjects, and by the Western Institutional Review Board (Puyallup, WA). All subjects were native English speakers with normal hearing and provided written informed consent.
 
 \paragraph{Hierarchical topographic factor analysis (HTFA).}
```
[the remark this answers is not in the document]
Comment (lucywowen) [reply paper/main.tex:1268]: yeah, they say no sub subheading only in the results section.

File: paper/supplementary_materials.tex (hunk 22 above)
Comment (lucywowen) [reply paper/supplementary_materials.tex:86]: done

File: paper/supplementary_materials.tex (hunk 23 above)
Comment (jeremymanning) [reply paper/supplementary_materials.tex:90]: bumping this

**Review by jeremymanning — CHANGES_REQUESTED** [review]
just one more change-- one missing comment to address in the supplemental materials.  also, i didn't check over the compiled PDFs, but you should do that carefully (e.g. make sure figures look good, references compile correctly, etc.).

File: paper/main.tex (hunk 1 above)
Comment (jeremymanning) [reply paper/main.tex]: still says "noise..."

File: paper/supplementary_materials.tex (hunk 16 above)
Comment (jeremymanning) [reply paper/supplementary_materials.tex]: still has a semicolon

File: paper/supplementary_materials.tex (hunk 17 above)
Comment (jeremymanning) [reply paper/supplementary_materials.tex]: still has a semicolon

**Pull request merged**